roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -62,9 +65,15 @@ def run(self, tmp, task_vars):
 
     def required_images(self, task_vars):
         deployment_type = get_var(task_vars, "deployment_type")
+        # FIXME(juanvallejo): we should handle gracefully with a proper error
+        # message when given an unexpected value for `deployment_type`.
```

Comment:
Yes, your PR will take care of ensuring this is set to some know value.

In this case, we accept/handle less than all the currently known values. We should not raise `KeyError` when `deployment_type` is e.g. `online` or some future new entry to the list of known values (we need not to force a lock-step update of the list of known values here and to the rest of openshift-ansible). Hmm.. or.... would it make sense to read from the YAML files that defines the known values?
Anyway, this is a discussion for a future PR and commenting here will probably be like writing to /dev/null :-)

Anyway, @juanvallejo FYI ^^^^